packages/legend-application-studio/src/stores/shared/PackageTreeUtils.ts

```diff
@@ -48,6 +48,10 @@ import {
   getElementRootPackage,
 } from '@finos/legend-graph';
 import { ExplorerTreeRootPackageLabel } from '../ExplorerTreeState.js';
+import { GAV_DELIMITER } from '@finos/legend-storage';
+
+const getArtifactIdFromProjectId = (projectId: string): string =>
```

Comment:
We should be cautious here in case there is not always an artifactId. 
Legacy projectId may still occur. 
We can use the same logic of ProjectDependency. Is its not legacy dependency then use the logic here else show full project id.